src/toasts/AnalyticsToast.tsx

```diff
@@ -75,53 +75,25 @@ const onLearnMorePreviouslyOptedIn = () => {
     });
 };
 
-const onUsageDataClicked = () => {
-    Analytics.showDetailsModal();
-};
-
 const TOAST_KEY = "analytics";
 
-const getAnonymousDescription = (): ReactNode => {
-    // get toast description for anonymous tracking (the previous scheme pre-posthog)
-    const brand = SdkConfig.get().brand;
+export function getPolicyUrl(): Optional<string> {
+    const policyUrl = SdkConfig.get("privacy_policy_url");
+    if (policyUrl) return policyUrl;
+
     const piwikConfig = SdkConfig.get("piwik");
```

Comment:
Could use a comment saying "try getting it from the legacy location" or something